### ISSUE TITLE
Downloads defaulting to ~/Downloads

### ISSUE THREAD

**Issue by kev12345** [sbussetti/deluge-siphon#22]
When I add torrents with deluge siphon they default to downloading to ~/Downloads even though my settings are set to download to a different location. I'm running the latest deluge daemon and web ui and have tried re-installing Raspbian and deluge from fresh.

Any ideas?

**Comment by sbussetti** [comment]
Where are you setting the alternate download location?  My extension checks for and uses the value of `download_location` which is set via the web ui in Settings -> Downloads -> Download to:

**Comment by sbussetti** [comment]
Also, what is the actual version number of Deluge you're running?

**Comment by kev12345** [comment]
Yes that's how mine is set. I confirmed the location in the core.conf

I downloaded a similar chrome extension to yours last night, 'remote
torrent adder'. This appears to works fine and downloads go into the
correct location.

To confirm, i have the latest deluge siphon (0.66.6)

**Comment by kev12345** [comment]
Deluge v1.3.10 installed

**Comment by kev12345** [comment]
Just a thought,

The deluge developers released some new instructions recently on how to
install deluge as a service rather than have it open using init.d and other
auto-start methods and so i reinstalled deluge using these instructions.
Having done this i noticed the deluge config (core.conf) now lives in
/var/lib/deluge/.config/deluge and I also believe there is another
core.conf elsewhere which my deluge isn't using.. Perhaps deluge siphon is
accessing that other core.conf for the download_location..?
As i said, 'remote torrent adder' is working fine for me so there's
obviously a way around it somehow...

**Comment by kev12345** [comment]
Just to clarify, when i say i reinstalled deluge i mean i installed it
afresh, after wiping my Pi and reinstalling Raspbian. So any core.conf
files existing are there as part of the install and aren't old historic
files..

**Comment by codierae** [comment]
Having the same issue - i have deluge core set to download to /downloads/ - yes it shows that way in webui - yet the siphon is defaulting to ~/Downloads - adding torrent files manually works fine n defaults to right folder - siphon fails - using deluge docker from linuxservers.io -- versions below
deluge: 1.3.13
libtorrent: 1.0.9.0
siphon 0.66.6

**Comment by oshotton** [comment]
I'm having the same issue here with deluge running in the LinuxServer Docker container. It always tries to download to the default location of /root/Downloads rather than the download location I have set in `Preferences -> Downloads -> Download To`. I have also confirmed that it is the same value in the `/config/core.conf` file. There is no `/var/lib/deluge` directory in this container though so it's not getting mixed up with other core.conf files.

Adding torrents manually and via the Remote Torrent Added Extension work as expected and download to the directory specified.

Deluge: 1.3.13
Siphon: 0.66.6
